probe_scraper/transform_pings.py

```diff
@@ -0,0 +1,140 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from datetime import datetime
+
+DATES_KEY = "dates"
+COMMITS_KEY = "git-commits"
+HISTORY_KEY = "history"
+NAME_KEY = "name"
+TYPE_KEY = "type"
+
+
+def pretty_ts(ts):
+    return datetime.utcfromtimestamp(ts).isoformat(' ')
+
+
+def make_ping_defn(definition, commit, commit_timestamps):
+    if COMMITS_KEY not in definition:
+        # This is the first time we've seen this definition
+        definition[COMMITS_KEY] = {
+            "first": commit,
+            "last": commit
+        }
+        definition[DATES_KEY] = {
+            "first": pretty_ts(commit_timestamps[commit]),
+            "last": pretty_ts(commit_timestamps[commit])
+        }
+    else:
+        # we've seen this definition, update the `last` commit
+        definition[COMMITS_KEY]["last"] = commit
+        definition[DATES_KEY]["last"] = pretty_ts(commit_timestamps[commit])
+
+    return definition
+
+
+def pings_equal(def1, def2):
+    return all((
+        def1.get(l) == def2.get(l)
+        for l in {
+            'bugs',
+            'data_reviews',
+            'description',
+            'notification_emails',
+            'include_client_id',
+            'send_if_empty',
+        }
+     ))
```

Comment:
Can't we just compare the dictionaries here?  The schema doesn't allow extra keys, so there shouldn't be anything about this that would be different from just `def1 == def2`.

Reply:
We probably can! I mostly copy-pasted the existing implementation. Will simplify.